eng/source-build-patches/0014-comment-performancesensitive.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/Utilities/Compiler/WellKnownTypeProvider.cs b/src/Utilities/Compiler/WellKnownTypeProvider.cs
+index b2e1b5876..7169c1500 100644
+--- a/src/Utilities/Compiler/WellKnownTypeProvider.cs
++++ b/src/Utilities/Compiler/WellKnownTypeProvider.cs
+@@ -84,7 +84,7 @@ public static WellKnownTypeProvider GetOrCreate(Compilation compilation)
+         /// <param name="fullTypeName">Namespace + type name, e.g. "System.Exception".</param>
+         /// <param name="namedTypeSymbol">Named type symbol, if any.</param>
+         /// <returns>True if found in the compilation, false otherwise.</returns>
+-        [PerformanceSensitive("https://github.com/dotnet/roslyn-analyzers/issues/4893", AllowCaptures = false)]
++        // [PerformanceSensitive("https://github.com/dotnet/roslyn-analyzers/issues/4893", AllowCaptures = false)]
```

Comment:
This fixes this error I was seeing:

```
roslyn-analyzers/artifacts/source-build/self/src/src/Utilities/Compiler/WellKnownTypeProvider.cs(87,10): error CS0122: 'PerformanceSensitiveAttribute' is inaccessible due to its protection level [roslyn-analyzers/artifacts/source-build/self/src/src/Microsoft.CodeAnalysis.Analyzers/Core/Microsoft.CodeAnalysis.Analyzers.csproj]
```

I have no idea what the consequences of this change might be.

Reply:
This error would be reported if the build incorrectly omitted [Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers/), which is a required analyzer package.